middlewares/account.middleware.js

```diff
@@ -113,12 +113,34 @@ async function addAccount(req, res, next) {
     next();
 }
 
+/**
+ * Updates an account that is specified by req.params.id
+ * @param {{params:{id: string}, body: *}} req 
+ * @param {*} res 
+ * @param {*} next 
+ */
+async function updateAccount(req, res, next) {
+    const account = await Services.Account.updateOne(req.params.id, req.body);
```

Comment:
Can you confirm that `updateOne` does return the account? If so, do we want to return the account information in the return status (as we do for other updates)?

Reply:
Edit: Disregard second question, I see that this just sends back req.body and not the account that we found

Reply:
- [x] ok